docs/src/main/asciidoc/podman.adoc

```diff
@@ -0,0 +1,58 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using Podman with Quarkus
+
+https://podman.io/[Podman] is a daemonless container engine for developing, 
+managing, and running OCI Containers on your Linux System or other OS. 
+
+Podman could be used the same way as Docker with `podman-docker` package. The big andwantage is that you could run Podman as regular Linux user. 
```

Comment:
```suggestion
Podman could be used the same way as Docker with the `podman-docker` package. The big advantage is that you could run Podman as regular Linux user. 
```

Reply:
I believe it's a ubntu / fedora package. So, it slightly different on mac / windows ,right?

Reply:
My linux machine run docker as a regular user, so, I would remove the last sentence.

Reply:
"advantage" sentence removed.